### DISCUSSION TITLE
Using vue-concurrency with Quasar

### DISCUSSION THREAD

**Discussion by realflight1** [quasarframework/quasar#9453] [General - Components / Directives / etc]
I am trying to use vue-concurrency with Quasar. Not sure if it belongs here or not, but I figured out it is probably related to my webpack configuration. I did open an issue in that project, see https://github.com/MartinMalinda/vue-concurrency/issues/43 .

At first I got errors related to `regeneratorRuntime` not being defined. Adding `regeneratorRuntime` using `webpack.ProvidePlugin` seems to fix this.

I am now getting a different error when using `usePipeTask` - "c.perform is not a function". I created a minimal example project to reproduce this with test code taken from vue-concurrency, see [this repository](https://github.com/realflight1/quasar-vue-concurrency-test). My changes over the default quasar project are minimal and appear in [this commit](https://github.com/realflight1/quasar-vue-concurrency-test/commit/94abf6e9f6a834d2fa2f35b39878a6af687d0a36).

I am wondering if it might be related to my configuration, or is it really an issue related only to vue-concurrency. Any ideas?

Thanks!

**Comment by ontwikkelfabriek** [comment]
Seems like a know bug ( obviously you already know that ):
https://github.com/MartinMalinda/vue-concurrency/issues/43

Please keep us posted if/when the issue has been resolved!

**Reply by realflight1** [reply]
Yes, surprisingly it's not my code :laughing:
